--- mediatomb-0.12.1.orig/src/string_converter.cc       2012-05-21 22:05:19.000000000 +0000
+++ mediatomb-0.12.1/src/string_converter.cc    2012-05-21 22:06:05.000000000 +0000
@@ -131,7 +131,7 @@
     //log_debug(("iconv: BEFORE: input bytes left: %d  output bytes left: %d\n",
     //       input_bytes, output_bytes));
 #if defined(ICONV_CONST) || defined(SOLARIS)
-    ret = iconv(cd, (const char**)input_ptr, &input_bytes,
+    ret = iconv(cd, (char**)input_ptr, &input_bytes,
             output_ptr, &output_bytes);
 #else
     ret = iconv(cd, input_ptr, &input_bytes,

